### PR TITLE
GSoC 2023: Aniket Agarwal Week 5

### PR DESCRIPTION
**Pull request by Aniket-debug**
Changes proposed in this pull request:
-  added start_vid and end_vid in one-to-one overload
-  added start_vid and end_vid columns in one-to-one docquiries
- fixed pgtap tests

@pgRouting/admins

### CODE REVIEW

**Comment by Aniket-debug** [comment]
@krashish8, @cvvergara I did very minimal changes and added start_vid and end_vid to one-to-one overload could you please tell me whether these changes would be acceptable or not?

**Comment by krashish8** [comment]
Yes, the output looks good

**Comment by Aniket-debug** [comment]
@krashish8 I did some changes in types_check.pg please look into it and let me know if I have missed something.

File: pgtap/ksp/ksp/types_check.pg [hunk 1]
```diff
@@ -18,26 +18,34 @@ along with this program; if not, write to the Free Software
...
 
 SELECT set_eq(
   $$SELECT  proargnames from pg_proc WHERE proname = 'pgr_ksp'$$,
-  $$SELECT  '{"","","","","directed","heap_paths","seq","path_id","path_seq","node","edge","cost","agg_cost"}'::TEXT[] $$
+  $$VALUES
+  ('{"","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[]),
+  ('{"","","","","directed","heap_paths","seq","path_id","path_seq",start_vid,end_vid,"node","edge","cost","agg_cost"}'::TEXT[])
```
Comment (krashish8) [inline pgtap/ksp/ksp/types_check.pg:35]: I think the records are duplicated. No use of duplicating the records when using set_eq - https://pgtap.org/documentation.html#set_eq

**Pull request merged**